abb_irb1600_support/urdf/irb1600_10_145_macro.xacro

```diff
@@ -0,0 +1,191 @@
+    </collision>
+  </link>
+  <link name="${prefix}link_2">
+    <visual>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://abb_irb1600_support/meshes/irb1600_10_145/visual/link_2.stl"/>
+      </geometry>
+      <material name="abb_orange"/>
+    </visual>
+    <collision>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://abb_irb1600_support/meshes/irb1600_10_145/collision/link_2.stl"/>
+      </geometry>
+      <material name="yellow"/>
+    </collision>
+  </link>
+  <link name="${prefix}link_3">
+    <visual>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://abb_irb1600_support/meshes/irb1600/visual/link_3.stl"/>
+      </geometry>
+      <material name="abb_orange"/>
+    </visual>
+    <collision>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://abb_irb1600_support/meshes/irb1600/collision/link_3.stl"/>
+      </geometry>
+      <material name="yellow"/>
+    </collision>
+  </link>
+  <link name="${prefix}link_4">
+    <visual>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://abb_irb1600_support/meshes/irb1600/visual/link_4.stl"/>
+      </geometry>
+      <material name="abb_orange"/>
+    </visual>
+    <collision>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://abb_irb1600_support/meshes/irb1600/collision/link_4.stl"/>
+      </geometry>
+      <material name="yellow"/>
+    </collision>
+  </link>
+  <link name="${prefix}link_5">
+    <visual>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://abb_irb1600_support/meshes/irb1600/visual/link_5.stl"/>
+      </geometry>
+      <material name="abb_orange"/>
+    </visual>
+    <collision>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://abb_irb1600_support/meshes/irb1600/collision/link_5.stl"/>
+      </geometry>
+      <material name="yellow"/>
+    </collision>
+  </link>
+  <link name="${prefix}link_6">
+    <visual>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://abb_irb1600_support/meshes/irb1600/visual/link_6.stl"/>
+      </geometry>
+      <material name="abb_orange"/>
+    </visual>
+    <collision>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://abb_irb1600_support/meshes/irb1600/collision/link_6.stl"/>
+      </geometry>
+      <material name="yellow"/>
+    </collision>
+  </link>
+  <!-- end of link list -->
+  <!-- joint list -->
+  <joint name="${prefix}joint_1" type="revolute">
+    <origin xyz="0 0 0" rpy="0 0 0"/>
+    <parent link="${prefix}base_link"/>
+    <child link="${prefix}link_1"/>
+    <axis xyz="0 0 1"/>
+    <limit lower="${radians(-180)}" upper="${radians(180)}" effort="0" velocity="${radians(180)}"/>
```

Comment:
Thanks for using `radians(..)` here.

The other variants don't use it yet, but it's good to start converting when adding new variants.